Portfolio/index.css

```diff
@@ -1,174 +1,123 @@
-console.log('Hello World');
 
-body {
-    font-family: Arial, sans-serif;
-    margin: 0;
-    padding: 0;
-    background-color: #f0f0f0;
-}
-/* Additional CSS styles */
-.exp img{
 
-width: 200px;
-height: 200px;
-border: 2px solid black;

```

Comment:
The comma selector is missing before the 'h4' selector at the beginning of line 14.